scripts/ci/copyright-exclude

```diff
@@ -0,0 +1,19 @@
+.clang-format
+.diff
+.md
```

Comment:
Should we use gitignore format? There is a parser for it https://pypi.org/project/gitignore-parser/

Reply:
We can consider it as an enhancement, either by switching to other tools to retrieve the list of files (`git ls-files` exlcude options only work with "ignored" or "other" files) or doing that ourselves. Note that this is just a file for the (extended) patterns we had in our workflow: https://github.com/model-checking/kani/blob/main/.github/workflows/format-check.yml#L16